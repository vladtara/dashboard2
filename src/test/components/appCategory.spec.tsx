import { render } from "@testing-library/react";
import AppCategory, { IAppCategoryProps } from "../../components/appCategory";

const props: IAppCategoryProps = {
  name: "Category Test",
  items: [
    {
      name: "App Test",
      icon: "bug_report",
      url: "#",
      displayURL: "test",
      newTab: false,
    },
    {
      name: "App Test",
      icon: "bug_report",
      url: "#",
      displayURL: "test",
      newTab: false,
    },
  ],
};

it("AppCategory snapshot test", () => {
  const { baseElement } = render(
    <AppCategory name={props.name} items={props.items} />,
  );

  expect(baseElement).toMatchSnapshot();
});
