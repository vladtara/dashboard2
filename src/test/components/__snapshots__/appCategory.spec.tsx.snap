// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppCategory snapshot test 1`] = `
<body>
  <div>
    <div
      class="sc-evZas csYxrp"
    >
      <h3
        class="sc-papXJ sc-crXcEl lhcacf ieLYyf"
      >
        Category Test
      </h3>
      <ul
        class="sc-jqUVSM egPCCj"
      >
        <li
          class="sc-kDDrLX OpKuo"
        >
          <a
            class="sc-dkzDqf gHJogt"
            href="#"
          >
            <div
              class="sc-hKMtZM kjGwCV"
            >
              <i
                class="sc-gsnTZi ctUcHQ"
              >
                bug_report
              </i>
            </div>
            <div
              class="sc-eCYdqJ bcsQqv"
            >
              <div
                class="sc-jSMfEi bkhPVp"
              >
                App Test
              </div>
              <p
                class="sc-gKXOVf EDUWD"
              >
                test
              </p>
            </div>
          </a>
        </li>
        <li
          class="sc-kDDrLX OpKuo"
        >
          <a
            class="sc-dkzDqf gHJogt"
            href="#"
          >
            <div
              class="sc-hKMtZM kjGwCV"
            >
              <i
                class="sc-gsnTZi ctUcHQ"
              >
                bug_report
              </i>
            </div>
            <div
              class="sc-eCYdqJ bcsQqv"
            >
              <div
                class="sc-jSMfEi bkhPVp"
              >
                App Test
              </div>
              <p
                class="sc-gKXOVf EDUWD"
              >
                test
              </p>
            </div>
          </a>
        </li>
      </ul>
    </div>
  </div>
</body>
`;
